### ISSUE TITLE
How can I use this library to read data from serial device in background when the terminalfragment is onPause?

### ISSUE THREAD

**Issue by uniqueram** [mik3y/usb-serial-for-android#496]
I have made a custom keyboard which is  a service and I need to read obtained data via keyboard using this library I could only get data to the keyboard when the terminalfragment is open and read data on UI but I need data when the terminalfragment is onPause.

**Comment by kai-morich** [comment]
you need a background service as used [here](https://github.com/kai-morich/SimpleUsbTerminal)

**Comment by uniqueram** [comment]
does this application not working in android version 13+?

**Comment by kai-morich** [comment]
did you try the background service? 
did do you observe a specific issue on android 13+?

**Comment by uniqueram** [comment]
yes, I did and worked on Android 11 but when I try on Android 13+ when data arrive from the serial device application crashes.

**Comment by kai-morich** [comment]
what kind of crash do you get?

**Comment by uniqueram** [comment]
I don't think the issue is like #494 I have tried that solution everything is working only when Data arrives from the serial device it gets crashed.

**Comment by kai-morich** [comment]
what kind of crash do you get?
